examples/eliza/tutorial_eliza.ipynb

```diff
@@ -0,0 +1,120 @@
+    "\n",
+    "### Install Dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "pip install git+https://github.com/asyml/forte.git@master\n",
+    "pip install stave==0.0.2.dev1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Start a Eliza pipeline service \n",
+    "\n",
+    "Run the following python script to start a pipeline service to process input queries based on Eliza rules."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from forte.pipeline import Pipeline\n",
+    "from forte.data.data_pack import DataPack\n",
+    "from forte.data.readers import RawDataDeserializeReader\n",
+    "from forte.processors.nlp import ElizaProcessor\n",
+    "\n",
+    "\n",
+    "def start_eliza_service(\n",
+    "    input_format: str = \"DataPack\", service_name: str = \"test_name\"\n",
+    ") -> None:\n",
+    "    \"\"\"\n",
+    "    Start a remote service for ElizaProcessor\n",
+    "    \"\"\"\n",
+    "    pipeline = Pipeline[DataPack]()\n",
+    "    pipeline.set_reader(RawDataDeserializeReader())\n",
+    "    pipeline.add(ElizaProcessor())\n",
+    "    pipeline.serve(input_format=input_format, service_name=service_name)\n",
+    "\n",
+    "\n",
+    "if __name__ == \"__main__\":\n",
+    "    start_eliza_service()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Visualize the chatbot in `stave`\n",
+    "\n",
+    "We will need `stave` to open a chatbot window for us to test the pipeline service that we just started in the previous step. `stave` is a fast, lightweight, extensible web-based text annotation and visualization tool designed to support a wide range of data types and NLP tasks. For more details, refer to https://github.com/asyml/stave.\n",
+    "\n",
+    "Open a new terminal window and run the following command:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "stave -s start -o -l"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we should see a browser window popped out which directs to a login page (http://localhost:8888/login) of `stave`. You can log in with default username (`admin`) and password (`admin`).\n",
+    "\n",
+    "After successfully logging into `stave`, we can navigate to the `All Projects` page. Click the `VIEW PROJECT` button under `Eliza` project. Then click `eliza.json` on the left side to enter the dialogue page where you can enter queries and get responses from Eliza chatbot."
```

Comment:
This sequence of actions would be better if have screenshots? 

Btw, if you upload the screenshots, you can upload them in your own repo, and use the link here (so the images won't make the main repo too large).